src/components/GameCard.tsx

```diff
@@ -75,10 +100,13 @@ export function GameCard(gameCardProps : GameCardProps ) {
             </div>
             <p>{gameCardProps?.dateAdded?.toDateString()}</p>
             <div>
-                <OpenLibraryModalButton text="Add to Library" onClick={addToLibrary}></OpenLibraryModalButton>
+                <Select
```

Comment:
What's the default value for our Select? If someone clicks the add button without selecting a library first, what do we submit?

I'm having a little trouble visualizing how this will all look/behave - could you attach a screenshot or screenrecording to the PR? We might want to get in the habit of doing that for our frontend PRs going forward.

Reply:
Updated to include that error logic. I'll post a screenrecording to the PR and discord shortly :)